spec/support/matchers/have_column.rb

```diff
@@ -8,7 +8,7 @@
     column && column.default.to_s == @default.to_s
   end
 
-  failure_message_for_should do |columns|
+  failure_message do |columns|
```

Comment:
You're right, I forgot to take it out of the changes; it was getting annoying to run the rspec tests and having deprecation warnings :cold_sweat: 

However, looking through RSpec 3 and Rspec 2.14, it seems that it is not backwards compatible.

- [RSpec 3](https://github.com/rspec/rspec-expectations/blob/master/lib/rspec/matchers/dsl.rb#L123-L125): The method `failure_message` is defined (with `failure_message_for_should` defined much later)
- [RSpec 2.14](https://github.com/rspec/rspec-expectations/blob/v2.14.5/lib/rspec/matchers/dsl.rb): It instantizes a [`RSpec::Matchers::DSL::Matcher`](https://github.com/rspec/rspec-expectations/blob/v2.14.5/lib/rspec/matchers/matcher.rb#L139-L141), where `failure_message_for_should` is defined, but no `failure_message`.

Therefore, it should not be backwards compatible.